scripts/zfs.sh

```diff
@@ -29,6 +29,7 @@ KMOD_ZCOMMON=${KMOD_ZCOMMON:-zcommon}
 KMOD_ZLUA=${KMOD_ZLUA:-zlua}
 KMOD_ICP=${KMOD_ICP:-icp}
 KMOD_ZFS=${KMOD_ZFS:-zfs}
+KMOD_FREEBSD=${KMOD_FREEBSD:-openzfs}
```

Comment:
If you want the script to load the in-tree kmod when run in-tree, you'll need to add the following line to `scripts/Makefile.am`.

```
export KMOD_FREEBSD=@abs_top_builddir@/module/zfs/openzfs
```

This will add the expanded environment variables to `scripts/common.sh` which is sourced by `scripts/zfs.sh` when run in-tree to get the in-tree paths.  `kldload` will also need to be able to accept the full kmod path.

You could probably even wrap this with `BUILD_LINUX / BUILD_FREEBSD` in the `Makefile.am` to only add the right platform specific entries.  Though adding both isn't a problem.

Reply:
Thanks! The correct path ended up being `@abs_top_builddir@/module/openzfs.ko`